.gitignore

```diff
@@ -151,4 +151,7 @@ allure-report/
 */scripts/local/
 
 # git conflict files
-*.orig
\ No newline at end of file
+*.orig
+
+# ignore tasks.json in .vscode
+!.vscode/tasks.json
```

Comment:
move this below .vscode block and update comment: # Ignore all files in .vscode except tasks.json